src/middlewares/credentials.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction} from "express";
+import { Credentials, CredentialsSchema } from '../models/credentials';
+
+export default (req: Request, res: Response, next: NextFunction) => {
+    const credentials: Credentials = req.body;
+    const validation = CredentialsSchema.validate(credentials);
+    
+    if (validation.error) {
+        return res.status(400).json(validation.error.details);
+    }
+    next();
```

Comment:
siempre deja un espacio antes y despues de un bloque IF, FOR, MAP, FIND, FILTER, REDUCE, etc